### ISSUE TITLE
How can you use ozow in a cart?

### ISSUE THREAD

**Issue by Codingapo** [wdtheprovider/ozpay#1]
Your coding skills are too much
but how can you use ozow payment integration on a checkout
Thanks for reading

**Comment by ubuntupunk** [comment]
Yep, me too. I see the test page will take you to ozow test which then provides a bunch of test server responses requiring further integration!

**Comment by Codingapo** [comment]
I FINALLY USED IT IN A CART

**Comment by ubuntupunk** [comment]
Please provide further instructions for doing this with a use case. The
current ozow prestashop plugin is failing.
